test/wallet-test.js

```diff
@@ -2869,4 +2869,343 @@ describe('Wallet', function() {
       assert.equal(claim.covenant.isClaim(), true);
     });
   });
+
+  describe('Create auction-related TX in advance', function () {
+    const network = Network.get('regtest');
+    const workers = new WorkerPool({ enabled });
+    const wdb = new WalletDB({ network, workers });
+    // This test executes a complete auction for this name
+    const name = 'satoshi-in-advance';
+    // There will be one bid. One from our wallet with a lockup and bid value made in advance
```

Comment:
nit: there are actually two bids in this test.

Reply:
Reworded the whole comment to make things clearer